code/game/machinery/vending.dm

```diff
@@ -803,6 +861,8 @@ IF YOU MODIFY THE PRODUCTS LIST OF A MACHINE, MAKE SURE TO UPDATE ITS RESUPPLY C
 					/obj/item/weapon/reagent_containers/food/snacks/donut = 14,/obj/item/weapon/storage/box/evidence = 6,/obj/item/device/flashlight/seclite = 6)
 	contraband = list(/obj/item/clothing/glasses/sunglasses = 2,/obj/item/weapon/storage/fancy/donut_box = 2)
 	premium = list(/obj/item/weapon/coin/antagtoken = 1)
+	special = list(/obj/item/weapon/gun/energy/laser/captain/scattershot = 10, /obj/item/weapon/claymore = 10, /obj/item/weapon/katana = 10, /obj/item/weapon/gun/energy/laser/captain = 10)
```

Comment:
What would be a better replacement weapon? It's pretty hard to get to the 1-3 or so cats on the station before the greytide, it should pay off to collect a whole set of cat teeth